File: src/main/java/no/nav/veilarbaktivitet/brukernotifikasjon/avslutt/SkalAvluttes.java
package no.nav.veilarbaktivitet.brukernotifikasjon.avslutt;

import lombok.Data;
import lombok.RequiredArgsConstructor;
import no.nav.veilarbaktivitet.person.Person;

import java.util.UUID;

@Data
@RequiredArgsConstructor
public class SkalAvluttes {
    private final String brukernotifikasjonId; // Samme som varselId
    private final Person.Fnr fnr;
    private final UUID oppfolgingsperiode;
}

File: src/main/kotlin/no/nav/veilarbaktivitet/brukernotifikasjon/avslutt/SkalAvluttes.kt
package no.nav.veilarbaktivitet.brukernotifikasjon.avslutt

import no.nav.veilarbaktivitet.person.Person
import java.util.*

data class SkalAvluttes(
    val brukernotifikasjonId: String,
    val fnr: Person.Fnr,
    val oppfolgingsperiode: UUID
)